packages/sit-onyx/src/composables/useRipple.ts

```diff
@@ -0,0 +1,86 @@
+import { computed, reactive, ref, type Ref } from "vue";
+import { distanceToFurthestCorner } from "../utils/math";
+
+export type RippleConfig = {
+  color: string;
+  terminateOnPointerUp: boolean;
+  container: Ref<HTMLElement | undefined>;
+};
+
+export type RippleInstance = {
+  left: string;
+  top: string;
+  radius: string;
+  time: number;
+  backgroundColor: string;
+  rippleId: string;
+  fadeIn: boolean;
+};
+
+export const useRipple = (config: Ref<RippleConfig>) => {
```

Comment:
Is it possible to also implement a reverse animation on mouse up? So the ripple "shrinks" back instead of hiding instantly

Reply:
I changed it like that .. I personally like the other version more, maybe you can decide with Jannick 🙂

Reply:
Yes I agree that it looks a little weird...I will check this with Jannick

Reply:
Sorry, it was a misunderstanding on my side. As discussed with Jannick, your previous behaviour is correct